Dockerfile.backwards

```
@@ -7,8 +7,7 @@ RUN echo "yaml https://s3-us-west-2.amazonaws.com/rosdep/base.yaml" > /etc/ros/r
 COPY . /opt/package
 WORKDIR /opt/package
 
-RUN apt-get install curl
-RUN curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | sudo apt-key add -
+RUN apt update || apt install -y curl && curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | apt-key add -
```

`curl --fail --location --show-error --silent` will:
* `--location`: follow redirects
* `--fail`: fail silently; dont output error document on failure
* `--silent`: Don't show progress bar
* `--show-error`: Shows error messages when used with `--silent`